lib/pascaltriangle.rb

```diff
@@ -0,0 +1,26 @@
+class Triangle
+
+  def initialize
+    @pascals_row = []
+  end
+
+  def pascal(row_number)
+    if row_number ==  0
+      @pascals_row << [1]
+    else
+      @pascals_row << (0..row_number).map { |i| binomial(row_number, i) }
```

Comment:
@zzokki81 from what I understand by looking at your code, this will always return the one row you requested. How and why are you using the `@pascals_row` variable?